diff --git a/src/microhttpd/connection.c b/src/microhttpd/connection.c
index abc622e..cd4e825 100644
--- a/src/microhttpd/connection.c
+++ b/src/microhttpd/connection.c
@@ -2136,10 +2136,6 @@
   }
 
   /* unescape URI *after* searching for arguments and log callback */
-  if (NULL != uri)
-    daemon->unescape_callback (daemon->unescape_callback_cls,
-                               connection,
-                               uri);
   connection->url = curi;
   return MHD_YES;
 }
